apps/src/blockly/addons/cdoXml.ts

```diff
@@ -10,39 +12,33 @@ import {
 // Once this has been done, all subsequent steps in the serialization use userCreated.
 const USER_CREATED_XML_ATTRIBUTE = 'usercreated';
 
-export default function initializeBlocklyXml(blocklyWrapper) {
-  // Clear xml namespace
-  blocklyWrapper.utils.xml.NAME_SPACE = '';
+export default function initializeBlocklyXml(
+  blocklyWrapper: BlocklyWrapperType
+) {
+  // Clear xml namespace. This property is readonly in Google Blockly.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (blocklyWrapper.utils.xml as any).NAME_SPACE = '';
 
   // Aliasing Google's domToBlock() so that we can override it, but still be able
   // to call Google's domToBlock() in the override function.
-  blocklyWrapper.Xml.originalDomToBlock = blocklyWrapper.Xml.domToBlock;
+  const originalDomToBlock = blocklyWrapper.Xml.domToBlock;
   // Override domToBlock so that we can gracefully handle unknown blocks.
   blocklyWrapper.Xml.domToBlock = function (
-    xmlBlock,
-    workspace,
-    parentConnection,
```

Comment:
We don't need these arguments anymore?

Reply:
they are not part of the [original api](https://developers.google.com/blockly/reference/js/blockly.xml_namespace.domtoblock_1_function.md) so they weren't doing anything